src/Tasks.UnitTests/CreateProperty_Tests.cs

```diff
@@ -47,7 +47,7 @@ public void CreateBlankProperty()
 
         /// <summary>
         /// Make sure that I can use the CreateProperty task to create a property
-        /// that has a parseable semicolon in it.
+        /// that has a parsable semicolon in it.
```

Comment:
Yeah, it was spelled both ways in the repo so I did the same think and did a search to find which was "correct"